src/renderer/modules/components/Tooltip.tsx

```diff
@@ -1,15 +1,42 @@
+import type { ObjectExports } from "../../../types";
 import type { HTMLAttributes } from "react";
 import { filters, getFunctionBySource, waitForModule } from "../webpack";
 
+const Aligns = {
+  BOTTOM: "bottom",
+  CENTER: "center",
```

Comment:
Does `WINDOW_CENTER` need to be added here?

Reply:
`WINDOW_CENTER` is for positions